### PR TITLE
improve spine preview

### PR DESCRIPTION
**Pull request by knoxHuang**
Re: #https://github.com/cocos/3d-tasks/issues/18622

### Changelog

*

image

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/i18n/zh/assets.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ module.exports = {
             animation: '动画',
             loop: '循环',
             timeScale: '速率',
+            premultipliedAlpha: 'Premultiplied Alpha',
+            useTint: 'Use Tint',
+            debugSlots: 'Debug Slots',
+            debugBones: 'Debug Bones',
+            debugMesh: 'Debug Mesh',
```
Comment (minggo) [inline editor/i18n/zh/assets.js]: Why these are english, and the above, such as `loop` is Chinese?

Comment (knoxHuang) [reply editor/i18n/zh/assets.js]: These are more intuitive in English, or we can temporarily use all English for now and replace them with Chinese property names later.

**Review by wuzhiming — APPROVED** [review]

File: editor/i18n/zh/assets.js (hunk 1 above)
Comment (minggo) [reply editor/i18n/zh/assets.js]: I think we should do it properly currently.

Comment (knoxHuang) [reply editor/i18n/zh/assets.js]: done

**Review by minggo — APPROVED** [review]

**Pull request merged**